Installation issue: gtkplus

### Steps to reproduce the issue

```console
$ spack spec -I gtkplus

Input spec
--------------------------------
 -   gtkplus

Concretized
--------------------------------
 -   gtkplus@3.24.29%gcc@8.5.0~cups~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]      ^at-spi2-atk@2.38.0%gcc@8.5.0~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]          ^at-spi2-core@2.48.3%gcc@8.5.0~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]              ^dbus@1.13.6%gcc@8.5.0~xml_docs build_system=autotools system-socket=default arch=linux-sle_hpc15-skylake_avx512
[+]                  ^docbook-xml@4.5%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]                  ^docbook-xsl@1.79.2%gcc@8.5.0 build_system=generic patches=a92c397 arch=linux-sle_hpc15-skylake_avx512
[+]                  ^libsm@1.2.3%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^libice@1.0.9%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libxtst@1.2.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^recordproto@1.14.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^atk@2.38.0%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^libffi@3.4.4%gcc@8.5.0 build_system=autotools patches=070b1f3 arch=linux-sle_hpc15-skylake_avx512
[+]      ^cairo@1.16.0%gcc@8.5.0+X+fc+ft+gobject+pdf+pic~png+shared~svg build_system=autotools patches=7c4da77 arch=linux-sle_hpc15-skylake_avx512
[+]          ^autoconf@2.69%gcc@8.5.0 build_system=autotools patches=35c4492,7793209,a49dd5b arch=linux-sle_hpc15-skylake_avx512
[+]          ^automake@1.16.5%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^fontconfig@2.14.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^font-util@1.4.0%gcc@8.5.0 build_system=autotools fonts=encodings,font-adobe-100dpi,font-adobe-75dpi,font-adobe-utopia-100dpi,font-adobe-utopia-75dpi,font-adobe-utopia-type1,font-alias,font-arabic-misc,font-bh-100dpi,font-bh-75dpi,font-bh-lucidatypewriter-100dpi,font-bh-lucidatypewriter-75dpi,font-bh-type1,font-bitstream-100dpi,font-bitstream-75dpi,font-bitstream-speedo,font-bitstream-type1,font-cronyx-cyrillic,font-cursor-misc,font-daewoo-misc,font-dec-misc,font-ibm-type1,font-isas-misc,font-jis-misc,font-micro-misc,font-misc-cyrillic,font-misc-ethiopic,font-misc-meltho,font-misc-misc,font-mutt-misc,font-schumacher-misc,font-screen-cyrillic,font-sun-misc,font-winitzki-cyrillic,font-xfree86-type1 arch=linux-sle_hpc15-skylake_avx512
[+]                  ^bdftopcf@1.1%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^fontsproto@2.1.3%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^libxfont@1.5.4%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                  ^mkfontdir@1.0.7%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                  ^mkfontscale@1.2.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^libfontenc@1.1.7%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^gperf@3.1%gcc@8.5.0 build_system=autotools patches=3dd36db arch=linux-sle_hpc15-skylake_avx512
[+]              ^util-linux-uuid@2.38.1%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^freetype@2.11.1%gcc@8.5.0+pic+shared build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^gmake@4.4.1%gcc@8.5.0~guile build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^libtool@2.4.7%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libx11@1.8.4%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^kbproto@1.0.7%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^xtrans@1.4.0%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxcb@1.14%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libpthread-stubs@0.4%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libxau@1.0.8%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libxdmcp@1.1.4%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                  ^libbsd@0.11.7%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^libmd@1.0.4%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^xcb-proto@1.15.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxext@1.3.3%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxrender@0.9.10%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^renderproto@0.11.1%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^m4@1.4.19%gcc@8.5.0+sigsegv build_system=autotools patches=9dc5fbd,bfdffa7 arch=linux-sle_hpc15-skylake_avx512
[+]              ^diffutils@3.9%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libsigsegv@2.14%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^pixman@0.42.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^python@3.11.6%gcc@8.5.0+bz2+crypt+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter+uuid+zlib build_system=generic patches=13fa8bf,b0615b2,ebdca64,f2fd060 arch=linux-sle_hpc15-skylake_avx512
[+]              ^expat@2.5.0%gcc@8.5.0+libbsd build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^gdbm@1.23%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libxcrypt@4.4.35%gcc@8.5.0~obsolete_api build_system=autotools patches=4885da3 arch=linux-sle_hpc15-skylake_avx512
[+]              ^openssl@3.1.3%gcc@8.5.0~docs+shared build_system=generic certs=mozilla arch=linux-sle_hpc15-skylake_avx512
[+]                  ^ca-certificates-mozilla@2023-05-30%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]              ^readline@8.2%gcc@8.5.0 build_system=autotools patches=bbf97f1 arch=linux-sle_hpc15-skylake_avx512
[+]              ^sqlite@3.43.2%gcc@8.5.0+column_metadata+dynamic_extensions+fts~functions+rtree build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^which@2.21%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^fixesproto@5.0%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^util-macros@1.19.3%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^fribidi@1.0.12%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^gdk-pixbuf@2.42.10%gcc@8.5.0~man~tiff~x11 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^libjpeg-turbo@3.0.0%gcc@8.5.0~ipo~jpeg8~partial_decoder+shared+static build_system=cmake build_type=Release generator=make arch=linux-sle_hpc15-skylake_avx512
[+]              ^cmake@3.27.9%gcc@8.5.0~doc+ncurses+ownlibs build_system=generic build_type=Release arch=linux-sle_hpc15-skylake_avx512
[+]              ^nasm@2.15.05%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libpng@1.6.39%gcc@8.5.0~ipo~pic build_system=cmake build_type=Release generator=make libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]          ^zlib-ng@2.1.5%gcc@8.5.0+compat+opt build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^gettext@0.22.4%gcc@8.5.0+bzip2+curses+git~libunistring+libxml2+pic+shared+tar+xz build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^bzip2@1.0.8%gcc@8.5.0~debug~pic+shared build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^libiconv@1.17%gcc@8.5.0 build_system=autotools libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxml2@2.10.3%gcc@8.5.0+pic~python+shared build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^ncurses@6.4%gcc@8.5.0~symlinks+termlib abi=none build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^tar@1.34%gcc@8.5.0 build_system=autotools zip=pigz arch=linux-sle_hpc15-skylake_avx512
[+]              ^pigz@2.7%gcc@8.5.0 build_system=makefile arch=linux-sle_hpc15-skylake_avx512
[+]              ^zstd@1.5.5%gcc@8.5.0+programs build_system=makefile compression=none libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]          ^xz@5.4.1%gcc@8.5.0~pic build_system=autotools libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]      ^glib@2.78.3%gcc@8.5.0~libmount~strip build_system=meson buildtype=release default_library=shared tracing=none arch=linux-sle_hpc15-skylake_avx512
[+]          ^elfutils@0.190%gcc@8.5.0~debuginfod+exeprefix+nls build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^pcre2@10.42%gcc@8.5.0~jit+multibyte build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^perl@5.38.0%gcc@8.5.0+cpanm+opcode+open+shared+threads build_system=generic patches=714e4d1 arch=linux-sle_hpc15-skylake_avx512
[+]              ^berkeley-db@18.1.40%gcc@8.5.0+cxx~docs+stl build_system=autotools patches=26090f4,b231fcc arch=linux-sle_hpc15-skylake_avx512
[+]      ^gobject-introspection@1.76.1%gcc@8.5.0~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]          ^bison@3.8.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^flex@2.6.3%gcc@8.5.0+lex~nls build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^findutils@4.9.0%gcc@8.5.0 build_system=autotools patches=440b954 arch=linux-sle_hpc15-skylake_avx512
[+]      ^inputproto@2.3.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^libepoxy@1.4.3%gcc@8.5.0+glx build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^glx@1.4%gcc@8.5.0 build_system=bundle arch=linux-sle_hpc15-skylake_avx512
[+]              ^mesa@23.2.1%gcc@8.5.0+glx+llvm+opengl~opengles+osmesa~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]                  ^glproto@1.4.17%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                  ^libunwind@1.6.2%gcc@8.5.0~block_signals~conservative_checks~cxx_exceptions~debug~debug_frame+docs~pic+tests+weak_backtrace~xz~zlib build_system=autotools components=none libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]                  ^libxt@1.1.5%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                  ^llvm@17.0.4%gcc@8.5.0+clang~cuda~flang+gold~ipo+libomptarget~libomptarget_debug~link_llvm_dylib+lld+lldb+llvm_dylib+lua~mlir+polly~python~split_dwarf~z3~zstd build_system=cmake build_type=Release compiler-rt=runtime generator=ninja libcxx=runtime libunwind=runtime openmp=runtime shlib_symbol_version=none targets=all version_suffix=none arch=linux-sle_hpc15-skylake_avx512
[+]                      ^binutils@2.41%gcc@8.5.0~gas+gold~gprofng+headers~interwork+ld~libiberty~lto~nls~pgo+plugins build_system=autotools compress_debug_sections=zlib libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]                      ^hwloc@2.9.1%gcc@8.5.0~cairo~cuda~gl~libudev+libxml2~netloc~nvml~oneapi-level-zero~opencl+pci~rocm build_system=autotools libs=shared,static arch=linux-sle_hpc15-skylake_avx512
[+]                          ^libpciaccess@0.17%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^libedit@3.1-20210216%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                      ^lua@5.3.6%gcc@8.5.0~pcfile+shared build_system=makefile fetcher=curl arch=linux-sle_hpc15-skylake_avx512
[+]                          ^unzip@6.0%gcc@8.5.0 build_system=makefile arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-data-dumper@2.173%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^swig@4.1.1%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]                  ^py-mako@1.2.4%gcc@8.5.0 build_system=python_pip arch=linux-sle_hpc15-skylake_avx512
[+]                      ^py-markupsafe@2.1.3%gcc@8.5.0 build_system=python_pip arch=linux-sle_hpc15-skylake_avx512
[+]      ^librsvg@2.56.2%gcc@8.5.0~doc build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^harfbuzz@7.3.0%gcc@8.5.0~graphite2~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]              ^icu4c@67.1%gcc@8.5.0 build_system=autotools cxxstd=11 arch=linux-sle_hpc15-skylake_avx512
[+]          ^rust@1.73.0%gcc@8.5.0~analysis+clippy~docs+rustfmt+src build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]              ^curl@8.4.0%gcc@8.5.0~gssapi~ldap~libidn2~librtmp~libssh~libssh2+nghttp2 build_system=autotools libs=shared,static tls=openssl arch=linux-sle_hpc15-skylake_avx512
[+]                  ^nghttp2@1.57.0%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^libgit2@1.7.0%gcc@8.5.0~curl~ipo+mmap+ssh build_system=cmake build_type=Release generator=make https=system arch=linux-sle_hpc15-skylake_avx512
[+]                  ^libssh2@1.11.0%gcc@8.5.0+shared build_system=autotools crypto=openssl patches=011d926 arch=linux-sle_hpc15-skylake_avx512
[+]                  ^pcre@8.45%gcc@8.5.0~jit+multibyte+pic+shared+static+utf build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]              ^rust-bootstrap@1.73.0%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]      ^libxi@1.7.6%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxfixes@5.0.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^xextproto@7.3.0%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^xproto@7.0.31%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^libxkbcommon@1.5.0%gcc@8.5.0~strip~wayland build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]          ^xkbdata@1.0.1%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[-]              ^xkbcomp@1.4.4%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[-]                  ^libxkbfile@1.0.9%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^meson@1.2.2%gcc@8.5.0 build_system=python_pip patches=0f0b1bd,ae59765 arch=linux-sle_hpc15-skylake_avx512
[+]          ^py-pip@23.1.2%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^py-setuptools@68.0.0%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^py-wheel@0.41.2%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]      ^ninja@1.11.1%gcc@8.5.0+re2c build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]          ^re2c@2.2%gcc@8.5.0 build_system=generic arch=linux-sle_hpc15-skylake_avx512
[+]      ^pango@1.50.13%gcc@8.5.0+X~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]          ^json-glib@1.6.6%gcc@8.5.0~strip build_system=meson buildtype=release default_library=shared arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxft@2.3.2%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^pkgconf@1.9.5%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]      ^shared-mime-info@1.10%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^intltool@0.51.0%gcc@8.5.0 build_system=autotools patches=ca9d656 arch=linux-sle_hpc15-skylake_avx512
[+]              ^perl-xml-parser@2.46%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                  ^perl-libwww-perl@6.68%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-clone@0.46%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-encode-locale@1.05%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-file-listing@6.04%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-html-parser@3.72%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                          ^perl-html-tagset@3.20%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-http-cookies@6.10%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-http-daemon@6.01%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                          ^perl-module-build-tiny@0.044%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                              ^perl-extutils-config@0.008%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                              ^perl-extutils-helpers@0.026%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                              ^perl-extutils-installpaths@0.012%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                              ^perl-module-build@0.4232%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-http-date@6.02%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-http-message@6.44%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                          ^perl-io-html@1.004%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-http-negotiate@6.01%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-lwp-mediatypes@6.02%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-net-http@6.22%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-try-tiny@0.31%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-uri@5.08%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                          ^perl-test-needs@0.002010%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]                      ^perl-www-robotrules@6.02%gcc@8.5.0 build_system=perl arch=linux-sle_hpc15-skylake_avx512
[+]      ^xrandr@1.5.0%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^libxrandr@1.5.3%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512
[+]          ^randrproto@1.5.0%gcc@8.5.0 build_system=autotools arch=linux-sle_hpc15-skylake_avx512

```


### Error message

Error message

```
/home/bgeltz/git/spack/lib/spack/env/gcc/gcc -Imodules/printbackends/libprintbackend-cups.so.p -Imodules/printbackends -I../spack-src/modules/printbackends -I. -I../spack-src -Igtk -I../spack-src/gtk -Igdk/x11 -I../spack-src/gdk/x11 -Igdk -I../spack-src/gdk -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/glib-2.78.3-5remxb4tztvp3qrvjcw4v4xkqouyn5av/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/glib-2.78.3-5remxb4tztvp3qrvjcw4v4xkqouyn5av/include/glib-2.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/glib-2.78.3-5remxb4tztvp3qrvjcw4v4xkqouyn5av/lib/glib-2.0/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/pcre2-10.42-ozl2xtpmbi37aw7u3joi6y5y7o6dnxtl/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libffi-3.4.4-4j3d756xyzmgplm2i4rzgc5ilbbeqbh3/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/glib-2.78.3-5remxb4tztvp3qrvjcw4v4xkqouyn5av/include/gio-unix-2.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/zlib-ng-2.1.5-swyyj6lgsavo4svb7gqhdba22g4vjkb3/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/pango-1.50.13-x2k43b77bdyiqhdptyah5f34gecmfgia/include/pango-1.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/renderproto-0.11.1-hjfaeg7e7anai4jtrxn7e3rgd45e7aeh/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/kbproto-1.0.7-2uikb5fattqcoppmk46ww5bhouqk4xdq/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/xextproto-7.3.0-6kqersdm3ch4tm7cao6sfxsvf5kmfvs4/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/cairo-1.16.0-lbhaicx4yvwyjupaat3mbl2kstn7jncr/include/cairo -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/xproto-7.0.31-722mg2ybxe4zecuggkiapwttyyja7qzo/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/harfbuzz-7.3.0-l4uro3c35n2g6f7kst3y2nd7izmr2bcw/include/harfbuzz -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/fontconfig-2.14.2-sl5lgrlk3kezlaryaxnf7hjgh2bllvjy/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libx11-1.8.4-gvglkskcwfzesvfkh6zysrn5ywyqze2z/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/freetype-2.11.1-buyle2fhtigbqwxh5svjhishu65gvqfx/include/freetype2 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/pixman-0.42.2-6mqj7hcnt6pamllhsztrznhxhq4zmrg2/include/pixman-1 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxcb-1.14-hfkkmwxkwpwtlzkqso3avyllr37kslsc/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxext-1.3.3-det2ucrbtqafg5fe65hpmlosv7ov73ea/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxau-1.0.8-os2hvqsmuz7kos5ffkhabqowzsvvmjkf/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxdmcp-1.1.4-7ze4mfgpugppx7quz2bcxe3vys4i3dsh/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxml2-2.10.3-duqw6ztbmalwhgk5jtsaxiab63zhy4yf/include/libxml2 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libiconv-1.17-xy2i6z632w7rqcjsfg2i6ugblziergbx/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/bzip2-1.0.8-c6xqil6iw7su2brl2ntc2d4tsf6ui4ir/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/fribidi-1.0.12-5upb6e5yju7z6utsxnab4gcs4xl4hja6/include/fribidi -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxft-2.3.2-dbgcl33bshrjicssqh2m4fddhxnkhat5/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libpng-1.6.39-mvssvshhuybdzmunqvtlba7dwnqzl7t2/include/libpng16 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxrender-0.9.10-ci3uzccsand7do6gokwycpcgynwfuhju/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/gdk-pixbuf-2.42.10-s5hm66r4e3oprdm2xk64plt75rciz3uh/include/gdk-pixbuf-2.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libjpeg-turbo-3.0.0-5leimxq2tvz3ckkvvn5enerigsrfu6rq/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/atk-2.38.0-nmfyw6gqjckbpz7c33qroaxldtgkgt27/include/atk-1.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libepoxy-1.4.3-qvotz74xr73iyklm5tjgci4igqmp7h4h/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxi-1.7.6-urdwdwukgh6iby7j7tthmcacsmmhrm4f/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/inputproto-2.3.2-nzidzwpb66zo5bpmxnkn3uwrjugmg6js/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/fixesproto-5.0-54b4kde44667mjgyj25rzpgm7zyudtrn/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxfixes-5.0.2-iwwttmvzj5beomoqyuyheug3zbbltv7z/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/at-spi2-atk-2.38.0-6h5nsbihipcuy4cpo4fqmvxj7ea6bm4s/include/at-spi2-atk/2.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/recordproto-1.14.2-4gjqpps2cfv67vxvb3qhcyxpg6uzv5ls/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/dbus-1.13.6-6at5wpljz3aqisein457t262dfsvxv4a/include/dbus-1.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/dbus-1.13.6-6at5wpljz3aqisein457t262dfsvxv4a/lib/dbus-1.0/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/at-spi2-core-2.48.3-myqno2n6llk4tpache5bpwjrnb5g5dgw/include/at-spi-2.0 -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxtst-1.2.2-7zxuopj5n7zax3aqdelhmlzgr3vftedi/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/libxrandr-1.5.3-lujhivymzvqauvg77jq6334kptocizhl/include -I/home/bgeltz/git/spack/opt/spack/linux-sle_hpc15-skylake_avx512/gcc-8.5.0/randrproto-1.5.0-fx6pd4qxh7ogtgxpdeqyx7k3u3a5trrw/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O3 -DG_LOG_USE_STRUCTURED=1 -DGLIB_MIN_REQUIRED_VERSION=GLIB_VERSION_2_56 -DGLIB_MAX_ALLOWED_VERSION=GLIB_VERSION_2_58 '-DGTK_VERSION="3.24.29"' -D_GNU_SOURCE -DG_DISABLE_CAST_CHECKS -fPIC -pthread -DWITH_GZFILEOP -DGTK_COMPILATION -DGTK_DISABLE_DEPRECATION_WARNINGS -DGTK_PRINT_BACKEND_ENABLE_UNSUPPORTED -MD -MQ modules/printbackends/libprintbackend-cups.so.p/cups_gtkprintbackendcups.c.o -MF modules/printbackends/libprintbackend-cups.so.p/cups_gtkprintbackendcups.c.o.d -o modules/printbackends/libprintbackend-cups.so.p/cups_gtkprintbackendcups.c.o -c ../spack-src/modules/printbackends/cups/gtkprintbackendcups.c
../spack-src/modules/printbackends/cups/gtkprintbackendcups.c:34:10: fatal error: cups/cups.h: No such file or directory
 #include <cups/cups.h>
          ^~~~~~~~~~~~~
compilation terminated.
```



### Information on your system

* **Spack:** 0.22.0.dev0 (e138052853e092fe58cbcfb195135c980fc931c5)
* **Python:** 3.6.15
* **Platform:** linux-sle_hpc15-skylake_avx512
* **Concretizer:** clingo

### Additional information

[spack-build-02-build-out.txt](https://github.com/spack/spack/files/14058670/spack-build-02-build-out.txt)
[spack-build-env.txt](https://github.com/spack/spack/files/14058689/spack-build-env.txt)


### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/gtkplus/package.py
+++ b/var/spack/repos/builtin/packages/gtkplus/package.py
@@ -15,6 +15,7 @@ class Gtkplus(MesonPackage):
 
     license("LGPL-2.0-or-later")
 
+    version("3.24.41", sha256="47da61487af3087a94bc49296fd025ca0bc02f96ef06c556e7c8988bd651b6fa")
     version("3.24.29", sha256="f57ec4ade8f15cab0c23a80dcaee85b876e70a8823d9105f067ce335a8268caa")
     version("3.24.26", sha256="2cc1b2dc5cad15d25b6abd115c55ffd8331e8d4677745dd3ce6db725b4fff1e9")
     version(
@@ -74,10 +75,11 @@ class Gtkplus(MesonPackage):
         return url.format(version.up_to(2), version)
 
     def patch(self):
-        # remove disable deprecated flag.
-        filter_file(
-            r'CFLAGS="-DGDK_PIXBUF_DISABLE_DEPRECATED $CFLAGS"', "", "configure", string=True
-        )
+        if self.spec.satisfies("@:3.24.35"):
+            # remove disable deprecated flag.
+            filter_file(
+                r'CFLAGS="-DGDK_PIXBUF_DISABLE_DEPRECATED $CFLAGS"', "", "configure", string=True
+            )
 
         # https://gitlab.gnome.org/GNOME/gtk/-/issues/3776
         if self.spec.satisfies("@3:%gcc@11:"):
@@ -95,7 +97,7 @@ class Gtkplus(MesonPackage):
         env.prepend_path("GI_TYPELIB_PATH", join_path(self.prefix.lib, "girepository-1.0"))
 
     def meson_args(self):
-        args = std_meson_args
+        args = []
 
         if self.spec.satisfies("platform=darwin"):
             args.extend(["-Dx11_backend=false", "-Dquartz_backend=true"])
@@ -103,6 +105,8 @@ class Gtkplus(MesonPackage):
         args.extend(
             ["-Dgtk_doc=false", "-Dman=false", "-Dintrospection=true", "-Dwayland_backend=false"]
         )
+
+        args.append("-Dprint_backends=file,lpr{0}".format(",cups" if "+cups" in self.spec else ""))
 
         return args
 

